cognite/client/config.py

```diff
@@ -6,6 +6,11 @@
 
 from cognite.client._version import __api_subversion__
 from cognite.client.credentials import CredentialProvider
+from cognite.client.utils._logging import (
+    _configure_logger_for_debug_mode,
+    _disable_debug_logging,
+    _is_debug_logging_enabled,
+)
```

Comment:
utils import on top is causing the following error:
```
ImportError: cannot import name 'global_config' from partially initialized module 'cognite.client.config' (most likely due to a circular import) (/Users/dmytrod/dev/cognite-sdk-python/cognite/client/config.py)
```
If I move the imports `global_config = GlobalConfig()` the error disappears.

Any recommendations on how to solve this issue @erlendvollset ?

Reply:
Easy option is to do local imports

Reply:
I see that it used to be a local import before my changes... maybe that's the reason why 😝